### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by anirudh-challa**
Fixed bugs from https://github.com/triagemd/keras-eval/issues/79 and https://github.com/triagemd/keras-eval/issues/80 

Also, changed setup so that there are no incompatible versions of packages.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,14 +5,16 @@
...
-    version='0.0.33',
+    version='0.0.34',
     description='An evaluation abstraction for Keras models.',
     author='Triage Technologies Inc.',
     author_email='ai@triage.com',
     url='https://www.triage.com/',
     packages=find_packages(exclude=['tests', '.cache', '.venv', '.git', 'dist']),
     install_requires=[
-        'Keras>=2.2.0',
+        'numpy==1.14.5',
```
Comment (adriaromero) [inline setup.py]: Why we need to fix `numpy` version? 🤔

File: setup.py [hunk 2]
```diff
@@ -5,14 +5,16 @@
...
+        'tensorflow==1.10',
```
Comment (adriaromero) [inline setup.py]: Recommended to import in `requirements.txt`.

File: tests/test_utils.py [hunk 3]
```diff
@@ -115,7 +115,8 @@ def test_compare_concept_dictionaries():
     concept_dict = [{'group': 'dog'}, {'group': 'cat'}, {'group': 'elephant'}]
     with pytest.raises(ValueError) as exception:
         utils.compare_group_test_concepts(concept_lst, concept_dict)
-    expected = 'The concept dictionary groups do not match the class labels'
+    expected = "('The following concepts are not present in the either the concept dictionary or among the " \
```
Comment (adriaromero) [inline tests/test_utils.py]: "The following concepts are not present in ~the~ either the concept dictionary..."

File: setup.py [hunk 4]
```diff
@@ -22,5 +24,6 @@
         'matplotlib',
         'sklearn',
         'keras-model-specs>=0.0.28',
+        'setuptools<=39.1.0'
```
Comment (jsalbert) [inline setup.py]: this is only needed to work with that tensorflow version, am I right?

Comment (anirudh-challa) [reply setup.py]: I think this is the setuptools version for both `1.10` and `1.11`. From my understanding, Setuptools is a package manager which should help in getting the correct versions of packages. 

If this line isn't there you get a mismatch of setuptools version and tensorflow version.

File: keras_eval/eval.py [hunk 5]
```diff
@@ -33,7 +32,7 @@ def __init__(self, **options):
 
         for key, option in self.OPTIONS.items():
             if key not in options and 'default' not in option:
-                raise ValueError('missing required option: %s' % (key, ))
+                raise ValueError('missing required option: %s' % (key,))
```
[the remark this answers is not in the document]
Comment (anirudh-challa) [reply keras_eval/eval.py]: It a consequence of `script/autolint`

File: keras_eval/utils.py [hunk 6]
```diff
@@ -165,10 +165,15 @@ def compare_group_test_concepts(test_concepts_list, concept_dict):
         True, if there are no repeat concepts, else raises error
     '''
     concept_group_lst = get_concept_items(concept_dict, key="group")
-    if set(concept_group_lst) == set(test_concepts_list):
-        return True
+
+    different_concept_set = set(concept_group_lst).symmetric_difference(set(test_concepts_list))
```
[the remark this answers is not in the document]
Comment (anirudh-challa) [reply keras_eval/utils.py]: Doing `set(concept_group_lst) - set(test_concepts_list)` will give you only the set elements that are in `concept_group_lst` but not in `test_concepts_list`, but doing `symmetric_differce` gives you all the elements are not in the interesection of the 2 sets.

File: setup.py (hunk 1 above)
Comment (anirudh-challa) [reply setup.py]: If not there is an incompatible version that gets installed.

File: setup.py (hunk 2 above)
Comment (jsalbert) [reply setup.py]: what was the reason to move it here?

Comment (anirudh-challa) [reply setup.py]: I thought it was maybe because it was not a direct dependency but indirect dependency.

File: setup.py (hunk 1 above)
Comment (adriaromero) [reply setup.py]: I'm just curious what's the package that is incompatible?

Comment (anirudh-challa) [reply setup.py]: I think `tensorflow 1.10` if I remember right as the default version of numpy is higher than tf can  take.

**Pull request merged**